src/lib/select/select.md

```diff
@@ -1,111 +1,161 @@
 `<mat-select>` is a form control for selecting a value from a set of options, similar to the native
 `<select>` element. You can read more about selects in the
 [Material Design spec](https://material.google.com/components/menus.html). It is designed to work
-inside of an `<mat-form-field>` element.
+inside of a [`<mat-form-field>`](https://material.angular.io/components/form-field/overview)
+element.
+
+To add options to the select, add `<mat-option>` elements to the `<mat-select>`. Each `<mat-option>`
+has a `value` property that can be used to set the value that will be selected if the user chooses
+this option. The content of the `<mat-option>` is what will be shown to the user.
 
 <!-- example(select-overview) -->
 
-### Simple select
+### Getting and setting the select value
 
-In your template, create an `mat-select` element. For each option you'd like in your select, add an
-`mat-option` tag. Note that you can disable items by adding the `disabled` boolean attribute or
-binding to it.
+The select component is set up as a custom value accessor, so you can manipulate the select's value
+using any of the form directives from the core `FormsModule` or `ReactiveFormsModule`: `ngModel`,
```

Comment:
The select also supports setting the value via the `value` input which doesn't require forms.

Reply:
done and added example